packages/eslint-plugin/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "rootDir": "src",
     "outDir": "dist",
-    "module": "commonjs"
+    "module": "commonjs",
+    "target": "es2017"
```

Comment:
Do we want to target an older ES version?

Reply:
I used `es2017` to get around TS errors with `.entries()` and because it's also used in the webpack loader package. Let me know if that's a problem.

Reply:
According to https://node.green/, our minimum version supports all the features we need, so all good